.gitignore

```diff
@@ -25,3 +25,6 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+.idea
```

Comment:
I don't think it's a good idea to submit to repos gitignore files and dirs that do not belongs to project itself. You can always set them in your user gitignore. Some devs use idea, some vscode, some sublime, etc. Let's keep project clean!